grafana/block_verification.json

```
@@ -101,6 +128,7 @@
       },
       "yaxes": [
         {
+          "$$hashKey": "object:84",
           "format": "short",
```

```suggestion
          "format": "none",
```

Shows the entire height instead of e.g. "1.3 M"

---

Aha, I was looking for this setting!